src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,84 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private LinkedList<Entry<K, V>>[] table;
+    private int size;
 
+    @SuppressWarnings("unchecked")
+    public MyHashMap() {
+        table = (LinkedList<Entry<K, V>>[]) new LinkedList<?>[INITIAL_CAPACITY];
```

Comment:
The unchecked cast warning persists here due to the creation of a generic array. Consider using `LinkedList<?>` to suppress this warning, or explore other ways to handle generic arrays in Java.